app/models/chargeback/rates_cache.rb

```diff
@@ -0,0 +1,24 @@
+class Chargeback
+  class RatesCache
+    def get(perf)
+      @rates ||= {}
+      @rates[perf.hash_features_affecting_rate] ||=
+        begin
+          prefix = tag_prefix(perf)
```

Comment:
@isimluk is this same as 
```
  prefix = Chargeback.report_cb_model(self.class.name).underscore
```
?

We was deriving `prefix` from chargeback class, (basically is report for VM, Projects or Contr. )